### ISSUE TITLE
Feature Request: Report Basal Total

### ISSUE THREAD

**Issue by marionbarker** [ivalkou/freeaps#68]
There are two screens where seeing the scheduled basal total for the day would be useful. 
1. Basal Profile to save on pump
2. Autotune screen

**Comment by bjornoleh** [comment]
Agreed. I think the total daily basal was removed from Loop Dev recently. Some of the feedback then was related to the loss of a sanity check on settings. Some are also used to consider the split between total daily basal and bolus insulin, although this may be a little arbitrary with an APS.
